packages/blocks/src/root-block/widgets/pie-menu/base.ts

```diff
@@ -61,21 +61,13 @@ export interface IPieCommandNode extends IPieBaseNode {
 // Open a submenu
 export interface IPieSubmenuNode extends IPieBaseNode {
   type: 'submenu';
-  role: 'default' | 'color-picker';
+  role: 'default' | 'color-picker' | 'command';
+  action?: ActionFunction;
   children: Array<IPieNonRootNode>;
-  // TODO
   openOnHover?: boolean;
   timeoutOverride?: number;
 }
 
-export type IPieNodeWithAction = IPieCommandNode; // | IPieToggleNode;
-
-export type IPieNonRootNode = IPieCommandNode | IPieColorNode | IPieSubmenuNode;
-// | IPieToggleNode;
-
-export type IPieNode = IPieRootNode | IPieNonRootNode;
-
-// For a color picker sub menu
 export interface IPieColorNode extends IPieBaseNode {
```

Comment:
Although this is an interface, we usually don't prefix with the letter 'I'. `PieColorNode` is fine.